apps/root/src/services/accountService.ts

```diff
@@ -22,12 +22,15 @@ export default class AccountService implements IAccountService {
     return this.user?.wallets || [];
   }
 
-  async setActiveWallet(wallet: string): Promise<void> {
-    this.activeWallet = find(this.user?.wallets || [], { address: wallet.toLowerCase() })!;
-    const provider = await this.activeWallet.getProvider();
+  setActiveWallet(wallet: string): void {
+    const newActiveWallet = find(this.user?.wallets || [], { address: wallet.toLowerCase() })!;
+    this.activeWallet = newActiveWallet;
+  }
 
-    await this.web3Service.connect(provider, undefined, undefined, this.activeWallet.type === WalletType.embedded);
-    return;
+  async setAndConnectActiveWallet(wallet: string): Promise<void> {
```

Comment:
what would be the difference between setting the active wallet and connecting the active wallet? Both should be the same action